addon/services/log.js

```diff
@@ -471,7 +475,15 @@ export default Service.extend(Evented, {
     @private
   */
   _storeToApplicationLog(category, message, formattedMessage) {
-    if (!this.get('enabled') ||
+    let isSkippedMessage = false;
+
+    skippedErrorsMessages.forEach(skippedMessage => {
+      if (category.name === skippedMessage.name && message.includes(skippedMessage.message)) {
```

Comment:
The `includes` method has [some gaps in browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#Browser_compatibility), I suggest using something more classic, for example, [`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf).